### PR TITLE
Improve FastAPI tutorial with Overlays usage

### PR DESCRIPTION
**Pull request by ChristopheDujarric**

### CODE REVIEW

File: src/_guides/bump-sh-tutorials/fastapi.md [hunk 1]
```diff
@@ -31,3 +33,31 @@ The following assumes your local machine is configured with Python and FastAPI,
...
+```shell
+bump overlay openapi.json overlays.json > openapi.updated.json
+```
+
+3. Deploy the resulting file to Bump.sh
+  ```shell
+  bump deploy openapi.updated.json \
+    --doc my-documentation-name \
+    --token my-documentation-token
+  ```
```
Comment (paulRbr) [inline src/_guides/bump-sh-tutorials/fastapi.md:59]: Real question: should we push for the 2 step process of `bump overlay` + `bump deploy` or should we mostly suggest to use the combined version of `bump deploy --overlay` in a single step?

File: src/_guides/bump-sh-tutorials/fastapi.md [hunk 2]
```diff
@@ -31,3 +33,31 @@ The following assumes your local machine is configured with Python and FastAPI,
...
+
+> Note: The diff calculation that Bump.sh performs to display the [API changelog](/help/changes-management/changelog) is done when the resulting file is deployed.
```
Comment (paulRbr) [inline src/_guides/bump-sh-tutorials/fastapi.md:61]:
```suggestion
> Note: The diff computation performed by Bump.sh to display the [API changelog](/help/changes-management/changelog) is done when the resulting file is deployed.
```

File: src/_guides/bump-sh-tutorials/fastapi.md [hunk 3]
```diff
@@ -31,3 +33,31 @@ The following assumes your local machine is configured with Python and FastAPI,
...
+> If you need to calculate the diff before applying Overlays (for instance when your Overlay document filters out some information), consider using the [`bump diff`](/help/continuous-integration/cli#bump-diff-file) command and the [GitHub Action](/help/continuous-integration/github-actions).
```
Comment (paulRbr) [inline src/_guides/bump-sh-tutorials/fastapi.md:62]:
```suggestion
> If you need to compute the diff before applying Overlays (for instance when your Overlay document filters out some information), consider using the [`bump diff`](/help/continuous-integration/cli#bump-diff-file) command and the [GitHub Action](/help/continuous-integration/github-actions).
```

File: src/_guides/bump-sh-tutorials/fastapi.md (hunk 1 above)
Comment (ChristopheDujarric) [reply src/_guides/bump-sh-tutorials/fastapi.md]: Good point, I've updated the guide!

File: src/_guides/bump-sh-tutorials/fastapi.md [hunk 4]
```diff
@@ -31,3 +33,26 @@ The following assumes your local machine is configured with Python and FastAPI,
...
+
+FastAPI lets you extend documents, [using Python](https://fastapi.tiangolo.com/how-to/extending-openapi/). 
+
+You can also accomplish this using OpenAPI's [Overlay specification](https://github.com/OAI/Overlay-Specification/blob/main/versions/1.0.0.md), which can be convenient in all use cases where you do not want to change your Python code base (e.g. when involving Technical Writers or Product Managers in writing docs).
+
+The Bump.sh CLI supports Overlays. Technically, the `bump overlay` command will output a modified version of the `[DEFINITION_FILE]` (an OpenAPI or AsyncAPI document) by applying the operations described in the `[OVERLAY_FILE]` Overlay file to the original API document.
+
+1. Create your Overlay document (see example use cases in [Augmenting Generated OpenAPI Documents with Filters & Overlays](/guides/openapi/augmenting-generated-openapi)), and name it `overlays.json` (YAML and JSON are both supported and input and outputs of the CLI).
+
+2. Apply your Overlay document to the original API document and deploy the resulting file to Bump.sh
```
Comment (paulRbr) [inline src/_guides/bump-sh-tutorials/fastapi.md:49]: can you split back the step in two please :pray:  (`bump overlay`, then `bump deploy`)

**Review by paulRbr — APPROVED** [review]
:+1

**Pull request merged**